macros/anaUltraScurve.py

```diff
@@ -115,6 +115,8 @@
     myT.Branch( 'scurve_h', scurve_h)
     chi2 = array( 'f', [ 0 ] )
     myT.Branch( 'chi2', chi2, 'chi2/F')
+    ndf = array( 'f', [ 0 ] )
```

Comment:
One could argue the "correct" way to do things is to have all the branches be floats and just cast all ints to floats. This is what I do in my analysis code. It is safer overall since integers are a subset of floats. Then you are also safe from funny type errors down the road since you know everything is a float down the road.